geopandas/io/arrow.py

```diff
@@ -388,9 +402,16 @@ def _read_parquet(path, columns=None, **kwargs):
     parquet = import_optional_dependency(
         "pyarrow.parquet", extra="pyarrow is required for Parquet support."
     )
+    filesystem = kwargs.get("filesystem")
+
+    if _is_fsspec_url(path) and filesystem is None:
```

Comment:
Can we raise an error if storage_options is not None if filesystem is not None? As in that case, we will now silently ignore storage_options, I think?